.github/workflows/push-1password-secrets-to-gha.yaml

```diff
@@ -9,6 +9,10 @@ on:
     branches:
       - "push-secrets-*"
 
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
```

Comment:
Hrrmmmm. Will we leave pulumi in a bad state if we don't let jobs finish?

Reply:
Flipped this over to not cancel in progress jobs. This should allow these workflows to queue up (with a max queue depth of 1)

>This means that there can be at most one running and one pending job in a concurrency group at any time. When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be pending. Any existing pending job or workflow in the same concurrency group, if it exists, will be canceled and the new queued job or workflow will take its place.

ref: https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs
ref: https://github.com/orgs/community/discussions/41518